backend/Data/Entities/Agency.cs

```diff
@@ -22,29 +17,22 @@ public class Agency : BaseEntity
         public string Code { get; set; }
```

Comment:
I would like it to have Code as well, wanted to see if my implementation of CodeEntity was acceptable first. Currently PropertyClassification doesn't support the Code property, it would need to if CodeEntity supported that field.